src/axom/sidre/examples/sidre_data_vs_metadata.cpp

```diff
@@ -225,12 +230,14 @@ int main(int argc, char* argv[])
   // Example 3: One-to-many Buffer to View relationships (view copy)
   // -----------------------------------------------------------------------
   //
-  // Create a buffer with a double array of length dat_size and initialize its
-  // data. Attach the buffer to two views, with data descriptions that vary in
-  // offset. Verify that the data associated with each view is correct.
+  // Create a copy of a view in one group in another group. Verify that the
```

Comment:
```suggestion
  // Create a copy of a view in another group. Verify that the
```
To me, "in one group in another group" jumped out as confusing.  Also, I think "in one group" is superfluous.

Reply:
how about "a copy of a group's view in a different group"?